index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://use.fontawesome.com/releases/v5.5.0/css/all.css"
+      integrity="sha384-B4dIYHKNBt8Bc12p+WXckhzcICo0wtJAoU8YZTY5qE0Id1GSseTk6S+L3BlXeVIU"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <div class="wrapper">
+      <!-- Start: Navbar-->
+      <nav>
+        <a class="logo" href="#">My Logo</a>
+        <i class="fas fa-bars"></i>
+      </nav>
+      <!-- End: Navbar-->
+
+      <!-- Start: Main Section-->
```

Comment:
- [ ] I noticed while reviewing your code that the spacing on the horizontal axis between your Toolbar and Header is too much when compared to the Figma design.

| Your Design  | Figma Design |
| ------------- | ------------- |
| <img width="224" alt="image" src="https://github.com/hamzaw9/portfolio/assets/52934128/be0005bb-14a6-4bfd-a512-f47393270125">  | <img width="317" alt="image" src="https://github.com/hamzaw9/portfolio/assets/52934128/3fe0bb95-b6e6-48e0-bc3f-caee6013b3e7">  |